dbt/include/teradata/macros/materializations/snapshot/strategies.sql

```diff
@@ -1,6 +1,6 @@
 
 {% macro teradata__snapshot_hash_arguments(args) -%}
-    HASHROW({%- for arg in args -%}
+    GLOBAL_FUNCTIONS.hash_md5({%- for arg in args -%}
```

Comment:
I suggest using the `dbt.hash` macro since the md5 function is configured differently in every environment.

Reply:
This would also require dbt.hash to be configurable. One proposal on how to do that can be found here: https://github.com/Teradata/dbt-teradata-utils/issues/2

Reply:
Regrettably, our `md5` implementation differs from the standard. Therefore, in our project, we have developed our own `teradata__hash` macro to align with the database configuration. Consequently, to accommodate this modification, we will also need to create a distinct implementation of `teradata__snapshot_hash_arguments` to cater to the custom `md5` setup.